index.html

```diff
@@ -0,0 +1,135 @@
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+<body class="body">
+
+    <header class="sign-up">
+        <div class="sign-up-description">
+            <p class="sign-up-text">
+                Already have an Intuit account?
+            </p>
+            <a href="#" class="sign-up-link">
+                I forgot my User ID or Password
+            </a>
+        </div>
+
+        <button
+            class="sign-in-button"
+        >
+            Sign In
+        </button>
+    </header>
+
+    <div class="main-content">
+
+   <div class="header">
+    <div class="logo">
+      <img src="asset/logo-intuit.png" alt="intuit logo">
+    </div>
+    <div class="icon-container">
+        <ul class="list">
+            <li class="list-item">
+                <div class="icon blue mint"></div>
+                <span class="icon-text">mint</span>
+            </li>
+
+            <li class="list-item">
+                <div class="icon green mint"></div>
+                <span class="icon-text">quickbooks</span>
+            </li>
+
+            <li class="list-item">
+                <div class="icon red mint"></div>
+                <span class="icon-text">turbotax</span>
+            </li>
+        </ul>
+    </div>
+   </div>
+
+   <div class="content-form">
+       <div class="description">
+        <h1 class="title-form">
+            Create an Intuit account
+        </h1>
+        <div class="text-container">
+            <p class="p">
+                One account for everything Intuit, including Mint. 
+            </p>
+            <a href="#" class="link">Learn more</a>
+        </div>
+
+       </div>
+        <form action="GET" class="form">
+
+            <div class="input-container">
+                <label for="email" class="label-input p">Email address</label>
+                <input type="email" id="email" class="input">
+            </div>
+
+            <div class="input-container">
+                <label for="phone" class="label-input p">Phone (recommended)</label>
+                <input type="tel" id="phone" class="input">
+            </div>
+
+            <div class="input-container">
+                <label for="password" class="label-input p">Password</label>
+                <input type="password" id="password" class="input">
+            </div>
+
+            <div class="input-container">
+                <input type="submit" value="Create account" class="button input">
+            </div>
+
+        </form>
+
+
+        <!-- policy -->
+
+        <div class="policy-container">
+            <p class="policy-text">
+              By clicking Create account, you agree to our
+              <a href="#" class="link">Terms</a>
+              and have read and acknowledge our 
+              <a href="#" class="link">Privacy Statement.</a> 
+            </p>
+        </div>
+
+
+
+
+
+        <!-- footer -->
+        <div class="footer">
```

Comment:
#### `HINT`

:bulb: Consider adding a `footer` element instead of as this is clearly the footer of this section. This gives the page more semantic meaning.

```suggestion
        <footer class="footer">
```